### PR TITLE
Single config

### PR DESCRIPTION
**Pull request by pconesa**
This PR should go with the rest:
https://github.com/scipion-em/scipion-pyworkflow/pull/53
https://github.com/scipion-em/scipion-app/pull/27
https://github.com/scipion-em/scipion-em/pull/51

### CODE REVIEW

File: motioncorr/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+scipion-em
```
Comment (azazellochg) [inline motioncorr/requirements.txt]: why this file is inside motioncorr folder? shouldn't it be outside?

Comment (azazellochg) [reply motioncorr/requirements.txt]: Also, I would put scipion-em requirement directly into setup.py instead of creating more files

Comment (pconesa) [reply motioncorr/requirements.txt]: You are right, my bad. It was meant to go to the root folder.

I can undo that part if you want.

**Comment by azazellochg** [comment]
All done already!

**Pull request merged**